recipes/isiscore/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "9.0.0" %}
+{% set build_number = "0" %}
+
+package:
+  name: isiscore
+  version: {{ version }}
+
+source:
+  # url: https://github.com/DOI-USGS/ISIS3/archive/{{ version }}.tar.gz
+  # sha256: 394ba889ceef5aaebc25e0e28f084209defb2c724e291ab86a587181bf095c7a
+  git_url: https://github.com/DOI-USGS/ISIS3.git
+  git_branch: new_pvl_core
```

Comment:
We cannot publish from a branch b/c they are not immutable. Can you ask upstream to mint a release? Worst case scenario, we can use a "branch tarball" format with a hash to freeze the state and publish it under CFEP-05 \*.

\* https://github.com/conda-forge/cfep/blob/main/cfep-05.md